### ISSUE TITLE
Next/previous navigation between discussion items

### ISSUE THREAD

**Issue by mitar** [peer/mind#207]
Migrating from [this comment](https://github.com/peer/mind/issues/205#issuecomment-277483616):

Allow meeting items to be processed and go to next item. See attached picture
![meeting items](https://cloud.githubusercontent.com/assets/6571828/22622167/4d05b2b2-eae8-11e6-8bf5-c41eaea35470.png)

cc @icmeyer

**Comment by mitar** [comment]
This might be tricky. Because it is unclear what next/previous is in general. If you came to a discussion from a meeting list you might want to go to the next one in the meeting, but if you are on the list of all discussions, the next discussion might be something else (all discussions list is ordered by time, while meetings have custom order).

One option to address this could be having a special "in the meeting mode" presentation view, which would be a special mode you project during a meeting and has a concept of a meeting and order of discussions (see #130).

Alternatively, we could remember from where you came to a discussion (direct link, meeting page, discussions list page) and then have next/previous based on that. The issue is that this can potentially confuse users why once next works in one way and the other time in another way. But it is also true that it will work often in the exactly the way they would want.

**Comment by mkanwal** [comment]
I strongly support eventually adding features geared towards improving the workflow for moderators.

One idea that was in the original conception of this platform was that there be a side navbar that lists all discussion items.  

e.g.,
![image](https://cloud.githubusercontent.com/assets/5432199/22631636/f65967fe-ebde-11e6-94e9-34811617218e.png)

**Comment by mitar** [comment]
I think this is to help a workflow during the meeting. Am I mistaken, @icmeyer? (Of course it could help also moderators, but for moderators I think the current Activity page is more useful.)

**Comment by icmeyer** [comment]
Yes, @mitar, this would be to assist in the facilitation of the meeting. A "meeting mode" is exactly what I am looking for. But the "meeting mode" could also include a sidebar to show what is coming up
